scipy/optimize/_linprog.py

```diff
@@ -575,6 +575,26 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
      success: True
            x: array([10., -3.]) # may vary
 
+    You can use the ``options`` parameter, e.g.,
+    to restrict the maximum number of iterations.
+
+    >>> res = linprog(
+            c,
+            A_ub=A,
+            b_ub=b,
+            bounds=[x0_bounds, x1_bounds],
+            options={'maxiter': 4}
+        )
```

Comment:
```suggestion
    You can use the ``options`` parameter, e.g.,
    to restrict the maximum number of iterations.

    >>> res = linprog(c, A_ub=A, b_ub=b, bounds=[x0_bounds, x1_bounds],
    ...               options={'maxiter': 4})
```
The `...` is needed to indicate line continuation in doctests.